MultimodalQnA/tests/test_compose_on_gaudi.sh

```diff
@@ -201,7 +201,24 @@ function validate_microservices() {
         "retriever-multimodal-redis" \
         "{\"text\":\"test\",\"embedding\":${your_embedding}}"
 
-    sleep 3m
+    echo "Wait for lvm-llava service to be ready"
```

Comment:
Did you check this on Gaudi? I think instead of `lvm-llava` we would want to check `tgi-llava-gaudi-server` (this is the container running the `huggingface/tgi-gaudi` image) and the tail of the log when it's ready looks like:
```
2024-12-16T20:00:03.491005Z  WARN text_generation_router: router/src/main.rs:372: Inferred max batch total tokens: 4096
2024-12-16T20:00:03.491013Z  INFO text_generation_router: router/src/main.rs:383: Setting max batch total tokens to 4096
2024-12-16T20:00:03.491020Z  INFO text_generation_router: router/src/main.rs:384: Connected
2024-12-16T20:00:03.491027Z  WARN text_generation_router: router/src/main.rs:398: Invalid hostname, defaulting to 0.0.0.0
```